tests/api/v1/endpoints/test_privacy_request_endpoints.py

```diff
@@ -1610,3 +1613,177 @@ def test_resume_privacy_request(
+        response = api_client.post(url, headers=auth_header, json=[{"mock": "row"}])
+        assert response.status_code == 400
+        assert (
+            response.json()["detail"]
+            == f"Invalid resume request: privacy request '{privacy_request.id}' status = in_processing."
+        )
+
+    def test_manual_resume_privacy_request_no_paused_location(
+        self, db, api_client, url, generate_auth_header, privacy_request
+    ):
+        auth_header = generate_auth_header(scopes=[PRIVACY_REQUEST_CALLBACK_RESUME])
+        privacy_request.status = PrivacyRequestStatus.paused
+        privacy_request.save(db)
+
+        response = api_client.post(url, headers=auth_header, json=[{"mock": "row"}])
+        assert response.status_code == 400
+        assert (
+            response.json()["detail"]
+            == f"Cannot resume privacy request '{privacy_request.id}'; no paused collection."
+        )
+
+    def test_resume_with_manual_input_collection_has_changed(
+        self, db, api_client, url, generate_auth_header, privacy_request
+    ):
+        """Fail if user has changed graph so that the paused node doesn't exist"""
+        auth_header = generate_auth_header(scopes=[PRIVACY_REQUEST_CALLBACK_RESUME])
+        privacy_request.status = PrivacyRequestStatus.paused
+        privacy_request.save(db)
+
+        privacy_request.cache_paused_step_and_collection(
+            ActionType.access, CollectionAddress("manual_example", "filing_cabinet")
+        )
+        response = api_client.post(url, headers=auth_header, json=[{"mock": "row"}])
+        assert response.status_code == 422
+        assert (
+            response.json()["detail"]
+            == "Cannot save manual rows. No collection in graph with name: 'manual_example:filing_cabinet'."
+        )
+
+    @pytest.mark.usefixtures(
+        "postgres_example_test_dataset_config", "manual_dataset_config"
+    )
+    def test_resume_with_manual_input_invalid_data(
+        self,
+        db,
+        api_client,
+        url,
+        generate_auth_header,
+        privacy_request,
+    ):
+        """Fail if the manual data entered does not match fields on the dataset"""
+        auth_header = generate_auth_header(scopes=[PRIVACY_REQUEST_CALLBACK_RESUME])
+        privacy_request.status = PrivacyRequestStatus.paused
+        privacy_request.save(db)
+
+        privacy_request.cache_paused_step_and_collection(
+            ActionType.access, CollectionAddress("manual_input", "filing_cabinet")
+        )
+        response = api_client.post(url, headers=auth_header, json=[{"mock": "row"}])
+        assert response.status_code == 422
+        assert (
+            response.json()["detail"]
+            == "Cannot save manual rows. No 'mock' field defined on the 'manual_input:filing_cabinet' collection."
+        )
+
+    @pytest.mark.usefixtures(
+        "postgres_example_test_dataset_config", "manual_dataset_config"
```

Comment:
If I'm understanding this correctly `manual_dataset_config` overlays `postgres_example_test_dataset_config` for these tests, and in test scenarios where we don't want to test manual input, we can overlay the same `dataset_config` as before?

Reply:
no, this is something Paul showed me, `pytest.mark.usefixtures` lets you separate the fixtures that you don't actually need to access within the function.   These tests still use both the postgres and manual datasets however.

Reply:
Got it, thanks